config/mock-data/addToCart/index.post.json

```diff
@@ -0,0 +1,4 @@
+{
```

Comment:
Does all of API calls have same structure of response? Usually its better to do it consistently 

`{
status: "success",
data:{
  "message": "Product added to cart successfully"
  }
}`

Reply:
Not in use, will update the structure with all the consistencies ASAP.